### ISSUE TITLE
'Killed' message during Training after epoch 0: 

### ISSUE THREAD

**Issue by renukaswamy** [songyouwei/ABSA-PyTorch#40]
Could someone please help me to come out of this issue:
I am using miniconda, tried to execute the sentiment analysis, i could see killed message. 
Here are the more details:

(base) renuk@renuk-ThinkPad-X1-Carbon-6th:~/ABSA-PyTorch$ python -c "import torch; print(torch.__version__)"
1.0.1.post2
(base) renuk@renuk-ThinkPad-X1-Carbon-6th:~/ABSA-PyTorch$ python
Python 3.6.2 |Anaconda, Inc.| (default, Oct  5 2017, 07:59:26) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
[4]+  Stopped                 python
(base) renuk@renuk-ThinkPad-X1-Carbon-6th:~/ABSA-PyTorch$ python train.py --model_name bert_spc --dataset restaurant --logdir bert_spc_logs
n_trainable_params: 109484547, n_nontrainable_params: 0
> training arguments:
>>> model_name: bert_spc
>>> dataset: restaurant
>>> optimizer: <class 'torch.optim.adam.Adam'>
>>> initializer: <function xavier_uniform_ at 0x7f65dd9719d8>
>>> learning_rate: 2e-05
>>> dropout: 0.1
>>> l2reg: 0.01
>>> num_epoch: 20
>>> batch_size: 64
>>> log_step: 5
>>> logdir: bert_spc_logs
>>> embed_dim: 300
>>> hidden_dim: 300
>>> bert_dim: 768
>>> pretrained_bert_name: bert-base-uncased
>>> max_seq_len: 80
>>> polarities_dim: 3
>>> hops: 3
>>> device: cpu
>>> model_class: <class 'models.bert_spc.BERT_SPC'>
>>> dataset_file: {'train': './datasets/semeval14/Restaurants_Train.xml.seg', 'test': './datasets/semeval14/Restaurants_Test_Gold.xml.seg'}
>>> inputs_cols: ['text_bert_indices', 'bert_segments_ids']
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
epoch:  0
Killed

**Comment by songyouwei** [comment]
It seems that you are running memory intensive BERT model on cpu.
This "killed" message is weird, this error is probably caused by insufficient memory?
